client/src/components/UserSidebar/UserSidebar.jsx

```diff
@@ -29,12 +39,16 @@ const UserSidebar = (props) => {
           subtitle={user ? user.email : ''}
           title={user ? user.username : ''}
         />
+        <CardActions>
```

Comment:
**Code Quality - Critical**
Only show this button when this profile belongs to the currently logged in user.